modules/builds-configuration-parameters.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * builds/build-configuration.adoc
+
+[id="builds-configuration-parameters_{context}"]
+= Build controller configuration parameters
+
+The `build.config.openshift.io/cluster` resource offers the following
+configuration parameters.
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`spec`
+|`BuildSpec` holds user-settable values for the build controller configuration.
+
+|`additionalTrustedCA`
+|A reference to a ConfigMap containing additional CAs that should be trusted for
+image pushes and pulls during builds. The namespace for this ConfigMap is
+`openshift-config`.
+
+|`buildDefaults`
+|Controls the default information for builds.
+
+|`buildOverrides`
+|Controls override settings for builds.
```

Comment:
can we find a way to structure this so it's obvious which fields are part of "buildDefaults" and which are part of "buildOverrides"?